### ISSUE TITLE
feature, how to change tray icon

### ISSUE THREAD

**Issue by ask-github** [lx-s/WinMute#9]
the light icon is not easy to find on the tray?
![图片](https://user-images.githubusercontent.com/72971953/149459118-7e552bf8-389a-4496-a5b1-161a874752f8.png)

**Comment by lx-s** [comment]
Good point. Current WinMute does not check whether the light or dark theme is activated and assumes the dark theme.
This'll go on my list

**Comment by lx-s** [comment]
Hey there, as in other issues: Sorry for the delay... I just started working on WinMute the other day again.
Due to my job and my family taking a large chunk of my time it really takes a while to get to these hobby projects...

I've attached a preview 1.7.0 for you here: [WinMute_1.7.0_prev.zip](https://github.com/lx-s/WinMute/files/8094825/WinMute_1.7.0_prev.zip). It contain s a dark tray icon as well as a detection logic when the system theme changes from bright to dark.

Since I've upgrade to Visual Studio 2022 recently, you might need to install this package from Microsoft: https://docs.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170

**Comment by ask-github** [comment]
@lx-s i find that this app [WinMute_1.7.0_prev.zip after run some days. it can't mute win10 on screen lock.
